### PR TITLE
Messages page

### LINKED ISSUE #814: Funcionalidades de Mensagens
## Feature Description

Na área do usuário tem um menu chamado MENSAGENS e um badget com o numero de mensagens novas.

- Apresentar todos os emails/notificações enviadas para o usuário
- Será uma forma de registrar toda a comunicação da plataforma com seus usuários.
- A lista apresentar o assunto, data e hora de envio
- Registros em negrito como NÃO LIDOS
- Ao clicar no registro apresentar o conteúdo: Assunto, mensagem, data de envio e um botão de FECHAR e MARCAR COMO NÃO LIDO.

O backend foi criado na PR #696  

---------------

## Acceptance criteria

* 

## Implementation Brief

*

### PR DESCRIPTION
**Pull request by luizeboli**
## Summary

Resolves #814 
https://www.loom.com/share/674484fe3c164ba9af39047bc11eed04

## Relevant technical choices

* Had to create a message seeder so we can use fake data in cypress tests

## QA Steps

1. Update your database: `npm run migration:refresh -- --seed`
2. Login with `sabiatestinge2e@gmail.com` user
3. Click user dropdown
4. Click `Mensagens` link
5. Check if you can see a list of messages, open a message and paginate

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code passes the linting.
- [x] My code has proper inline documentation.
- [x] I have manually tested this PR.

### CODE REVIEW

File: packages/web/components/SafeHtml/SafeHtml.js [hunk 1]
```diff
@@ -32,20 +33,26 @@ const SafeHtml = ({ html, as, allowedTags, allowedAttrs }) => {
 		cleanedHTML = `<${as}>${cleanedHTML}</${as}>`;
 	}
 
-	return parse(cleanedHTML);
+	if (shouldParse) {
+		cleanedHTML = parse(cleanedHTML);
+	}
```
Comment (luizeboli) [inline packages/web/components/SafeHtml/SafeHtml.js:38]: Added a `shouldParse` option, so we can choose to have only the html string

File: packages/web/pages/user/my-account/my-services.js [hunk 2]
```diff
@@ -48,7 +48,6 @@ const MyServices = ({
 				totalPages: initialTotalPages,
 				totalItems: initialTotalItems,
 			},
-			revalidateOnMount: true,
```
Comment (luizeboli) [inline packages/web/pages/user/my-account/my-services.js:51]: This was triggering a duplicated request.

We don't need o `revalidateOnMount` as we're passing data from `getInitialProps`

File: packages/web/pages/user/my-account/technologies.js [hunk 3]
```diff
@@ -26,7 +26,6 @@ const MyTechnologies = ({ initialTechnologies, user }) => {
 		(_, id) => getUserTechnologies(id),
 		{
 			initialData: initialTechnologies,
-			revalidateOnMount: true,
```
Comment (luizeboli) [inline packages/web/pages/user/my-account/technologies.js:29]: Same of the above case

File: packages/web/components/UserProfile/pages.js [hunk 4]
```diff
@@ -18,16 +18,22 @@ import {
  * @param {Function} t The function to translate the pages and sections titles.
  * @param {object} user The authenticated user object to filter pages.
  * @param {number} questions The authenticated user's number of unanswered questions.
+ * @param {number} messages The authenticated user's number of new messages.
```
[the remark this answers is not in the document]
Comment (luizeboli) [reply packages/web/components/UserProfile/pages.js]: Gotcha

**Review by mateus4k — APPROVED** [review]
💯

**Pull request merged**